ci-operator/step-registry/openshift-qe/orion/openshift-qe-orion-commands.sh

```
@@ -9,14 +9,15 @@ python --version
 pushd /tmp
 python -m virtualenv ./venv_qe
 source ./venv_qe/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: ./venv_qe/bin/activate was not specified as input (see shellcheck -x).

(SC1091)